screens/items.js

```diff
@@ -1,11 +1,12 @@
 import React from 'react';
-import { View, FlatList, StyleSheet, Text, TextInput } from 'react-native';
+import { View, FlatList, StyleSheet, Text, TextInput, TouchableOpacity } from 'react-native';
```

Comment:
It only took me a few hours to figure out I'd imported `TouchableOpacity` from the wrong place.

Reply:
That is a rough time. The buttons are centered, but the default increment is off the edge of the screen.
![image](https://user-images.githubusercontent.com/66502443/201394621-498b4542-8fa7-4bc8-b3a0-6361de7348a4.png)

Can we add a save button? I had a moment where I had no idea where to click after creating a new item.

Reply:
I changed the buttons to be relative again.  I couldn't find a way to set the icon size as a percentage, so the way I did it is sort of weird, but does it look better @MatthewWalstra?

Reply:
Currently it is saving whenever a value is changed, so a save button wouldn't do anything. It might make sense to add a floating done button?  I'm not sure it's worth doing too much now, but we can see if people in the user tests figure it out.